code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -348,6 +348,9 @@
 			missionobj.completed = 1
 			Commando.mind.objectives += missionobj
 
+			//Humans only
+			Commando.set_species("human")

```

Comment:
if you update their species you must update their dna. Put this right before Commando.dna.update_dna_identity() on line 335